### ISSUE TITLE
Colors not vivid

### ISSUE THREAD

**Issue by rbong** [gruvbox-community/gruvbox#9]
I am facing an issue where the colors on my terminal, URxvt, are not the same as the ones shown in the gallery.

![Uploading 2019-05-09-210928_1920x1080_scrot.png…]()

```
$ echo $TERM
rxvt-unicode-256color
```

I've tried the suggestions in [terminal-specific](https://github.com/morhetz/gruvbox/wiki/Terminal-specific), but none of them seem to work for me. 
Although running the `gruvbox_256palette.sh` does seem to make a slight difference, it is till not the same as that shown in the gallery.

Relevant lines of vimrc
```
 set t_8f=^[[38;2;%lu;%lu;%lum        " set foreground color
 set t_8b=^[[48;2;%lu;%lu;%lum        " set background color
 colorscheme gruvbox
 set t_Co=256                         " Enable 256 colors
 set termguicolors                    " Enable GUI colors for the terminal to get truecolor
```

Edit: I tried using the suckless-terminal (fresh install) but the colors rendered is the same as that shown above.

:robot: **This issue has been automatically copied from morhetz#279** :robot:

**Comment by ericdorsey** [comment]
@rbong 
Do you happen to use MobaXterm? If so, I was having the issue of muted / non vivid colors in your Issue title here (I can't see the image / the link doesn't work for me so I can't compare visually).

I was using an older version of MobaXterm, and when I would use `set termguicolors` the colors would be vivid, but the cursor in Vim would disappear. 

It appears this issue is resolved in MobaXterm v12.1 Build 4156.

```
set number
syntax on
set t_Co=256
set background=dark
set shiftwidth=4
set tabstop=4
set expandtab
set termguicolors "vivid colors

let g:gruvbox_italic=0
let g:gruvbox_contrast_dark = 'hard'
let g:gruvbox_colors = { 'bg0': ['#000000', 0] }
"let g:gruvbox_termcolors=16
colorscheme gruvbox
```

**Comment by rbong** [comment]
@ericdorsey the issue was copied from the original repository. Looking over this issue again, it seems like a problem with urxvt and other terminal color rendering, so I'm closing this issue for now.
